using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md

```diff
@@ -0,0 +1,21 @@
+# Anomaly Detection
+
+{% hint style="info" %}
+Anomaly Detection is currently in beta. Please read the documentation carefully.
+{% endhint %}
+
+Anomaly Detection is a governance tool which detects when cloud services significantly deviate from their projected spend. This feature, as well as [Forecasting](anomaly-detection.md#forecasting), are powered by the Anomaly Detection Container, which is enabled in Kubecost by default. Kubecost samples spend data from the last seven days to determine of a service's spend has become anomalous.

```

Comment:
It's probably better called the "modeling" container, as it's the home for all varieties of data-science/ML/etc. driven stuff.